packages/lit-dev-content/site/docs/ssr/client-api.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Lit SSR client API
+eleventyNavigation:
+  key: 🧪 Client API
+  parent: Server-side rendering
+  order: 3
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
+
+For more information about the Lit labs process, see [Lib Labs](/docs/libraries/labs/)
+
+</details>
+
+Lit SSR generates static HTML for the browser to parse and paint without any JavaScript. (Browsers that do not have support for Declarative Shadow DOM will require some JavaScript polyfill for Lit components authored to utilize the Shadow DOM.) For pages with static content, this is all that's needed. However, if the page content needs to be dynamic and respond to user interactions, it will need JavaScript to re-apply that reactivity.
+
+"Hydration" is the process having Lit re-associate the expressions of a Lit template with the  nodes they should update in the DOM, as well as adding reactivity and life cycles to Lit components.
```

Comment:
That's a good point. Since we do have the separate sections, perhaps it's better to forego defining hydration and just speak in terms of adding back reactivity. Then talk more of what "template hydration" and "component hydration" entails in the respective sections.